backend/rebutify/settings.py

```diff
@@ -94,6 +94,10 @@
         "rest_framework_simplejwt.authentication.JWTAuthentication",
     ],
     "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
+    # https://stackoverflow.com/a/47657610/19071246
+    # cursor pagination uses previous or next page links
+    "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.CursorPagination",
+    "PAGE_SIZE": os.getenv("PAGE_SIZE", 10),
```

Comment:
wrap with `int()` type conversion since os.getenv will return a string if the value is set